src/git-lfs/devcontainer-feature.json

```diff
@@ -13,8 +13,14 @@
             ],
             "default": "latest",
             "description": "Select version of Git LFS to install"
+        },
+        "autoPull": {
+            "type": "boolean",
+            "default": true,
+            "description": "Automatically pull LFS files when creating the container. If false, manually running 'git lfs pull' will pull lfs files."
         }
     },
+    "postCreateCommand": "/tmp/pull-git-lfs-artifacts.sh",
```

Comment:
Would adding this be an unexpected change for the user? 

1. Should we bump `major` version instead or 
2. Should we update `minor` version and set `autoPull` to `false` by default?

Reply:
Good point, I shouldn't have bumped the breakfix version.  I think the minor version.  From what I understand, it's expected behavior to auto pull (if you have git lfs already installed on your computer and do a `git clone`, you'd end up with the artifacts auto pulled).    This manifests differently in Codespaces since we do the clone first _before_ installing git-lfs, so there's never a change for that pull to happen. That's why I set it the default to true

Reply:
aside, this is why in the `scenarios.json` I need `GIT_LFS_SKIP_SMUDGE=1` in front of the git clone.  GitHub Actions must have git-lfs installed, so in CI it was automatically doing the pull when I cloned, which of course we don't want for this test :)